pages.id/linux/ruget.md

```diff
@@ -0,0 +1,12 @@
+# ruget
+
+> Alternatif untuk wget yang ditulis dalam Rust.
+> Informasi lebih lanjut: <https://github.com/ksk001100/ruget>.
+
+- Unduh konten URL ke file:
```

Comment:
I think `Mengunduh` is better for a verb.

Reply:
Yes, but that could cause potential conflict with the current tldr-pages style guide for English - see #8576 for details

Reply:
So, we should use `unduh` rather than `mengunduh`?